schema-tool/src/main/java/command/CassandraCommand.java

```diff
@@ -0,0 +1,87 @@
+package command;
+
+import com.scalar.db.config.DatabaseConfig;
+import core.SchemaOperator;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.Callable;
+import java.util.logging.Logger;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import utils.SchemaParser;
+
+@Command(name = "--cassandra", description = "Using Cassandra DB")
+public class CassandraCommand implements Callable<Integer> {
+
+  @Option(names = "-h", description = "Cassandra host IP", required = true)
+  String cassandraIP;
+
+  @Option(names = "-P", description = "Cassandra Port", defaultValue = "9042")
+  String cassandraPort;
+
+  @Option(names = "u", description = "Cassandra user", defaultValue = "cassandra")
+  String cassandraUser;
+
+  @Option(names = "-p", description = "Cassandra password", defaultValue = "cassandra")
+  String cassandraPw;
+
+  @Option(
+      names = "-n",
+      description =
+          "Cassandra network strategy, should be SimpleStrategy or NetworkTopologyStrategy")
+  String cassandraNetStrategy;
+
+  @Option(names = "-c", description = "Cassandra compaction strategy, should be LCS, STCS or TWCS")
+  String cassandraCompactStrategy;
```

Comment:
Should the value for this option be limited only to `LCS, STCS or TWCS` ? Maybe this can be done using an [enum](https://picocli.info/#_enum_types)